.htaccess

```diff
@@ -34,3 +35,7 @@ RedirectMatch 302 \
 RedirectMatch 302 \
 	^/the-cms-for-creators/?$ \
 	https://www.classicpress.net
+
+RedirectMatch 302 \
+	^/core-privacy-policy/?$ \
+	https://www.classicpress.net/privacy-policy
```

Comment:
Should we link to:
`https://www.classicpress.net/privacy-policy/#core-privacy-policy`?